src/carto.js

```diff
@@ -68,12 +75,74 @@ const getAttributeFeature = function (sym, feature, ly) {
   return addFunction(fnBody);
 };
 
+const getAlphaColor = function (color, opacity) {
+
+  if (color && opacity) {
+
+    var fn = `
```

Comment:
It is a string because is the tangram way of using dynamic properties, Tangram wraps every data passed...

For example, we have the next color property specified in the yaml config as
```yaml
layer_name:
  draw:
    polygons:
      color: function () {
          if (feature.size > 10) {return '#f00'}
          return '#0F0';
        }
```
Tangram parsed it and stores it in a config var as...
```
config.layers.layer_name.polygons.color: "function () {if (feature.size > 10) {return '#f00'} return '#0F0';}"
```

When tangram uses this variable it wraps the code like this...
```
  var feature = context.feature;
  
  fn = ${config.layers.layer_name.polygons.color};

  var val = fn();

  return val;

// it return a string like this, with a closure, so we can use feature inside our function in the configuration

  var feature = context.feature;
  
  fn = function () {if (feature.size > 10) {return '#f00'} return '#0F0';}"
  var val = fn();

  return val;
```

Then, using [this feature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function) generate a dynamic function (similar to eval code execution) allowing this kind of configuration.

What I am doing here is a double wrap, because we don't know the value, and it is a mix of two different configurations, so we have to lunch two function in one and it has to be the whole logic in there.